lib/client.js

```diff
@@ -504,7 +504,7 @@ async function lazyllhttp () {
     mod = await WebAssembly.compile(Buffer.from(llhttpWasmData || require('./llhttp/llhttp-wasm.js'), 'base64'))
   }
 
-  return await WebAssembly.instantiate(mod, {
+  return WebAssembly.instantiate(mod, {
```

Comment:
@ronag I imagine this is intentional to keep the stacktrace of the `lazyhttp` function, or it is safe to remove?

Reply:
I guess it should be safe, but don't really see any reason to do so. It's a cold path.